lightningd/channel/channel.c

```diff
@@ -275,9 +279,13 @@ static void handle_peer_funding_locked(struct peer *peer, const u8 *msg)
 		daemon_conn_send(&peer->master,
 				 take(towire_channel_normal_operation(peer)));
 	}
+
+	return peer_read_message(conn, &peer->pcs, peer_in);
```

Comment:
True, though returning the io_plan seemed like I was adding one less new abstraction.